Source/Scene/GlobeSurfaceShaderSet.js

```diff
@@ -1,18 +1,18 @@
 define([
-        '../Core/ClippingPlaneCollection',
         '../Core/defined',
         '../Core/destroyObject',
         '../Core/TerrainQuantization',
         '../Renderer/ShaderProgram',
         '../Scene/SceneMode',
```

Comment:
Unrelated to this PR, but for consistency this one should be changed to `./SceneMode`